app/Http/Controllers/LandingPageController.php

```diff
@@ -0,0 +1,21 @@
+<?php
+
+namespace App\Http\Controllers;
+
+use Illuminate\Http\Request;
+use App\Models\Post;
+use App\Models\User;
+class LandingPageController extends Controller
+{
+    public function index(){
+
+    
+
+        $posts = User::select('*')
+                ->join('posts', 'users.id', '=', 'posts.user_id')
+                ->get();
```

Comment:
It doesn't look good, you can do something like:
![image](https://user-images.githubusercontent.com/44239951/97212830-83d89d80-1786-11eb-823c-d0398b0ec06c.png)
Also, Take on your mind how important indentation is.

Reply:
And in your view, your foreach will look like: 
![image](https://user-images.githubusercontent.com/44239951/97213302-0c573e00-1787-11eb-88fd-04358cfb9245.png)